### ISSUE TITLE
Explicitly advise to use miniconda

### ISSUE THREAD

**Issue by adam-urbanczyk** [CadQuery/cadquery#691]
And to install it in a portable way

```cmd
bitsadmin /transfer myDownloadJob /download /priority normal https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe .
Miniconda3-latest-Windows-x86_64.exe /S /InstallationType=JustMe /D=%MINICONDA_DIRNAME%
set "PATH=%MINICONDA_DIRNAME%;%MINICONDA_DIRNAME%\\Scripts;%PATH%"
activate
```

**Comment by adam-urbanczyk** [comment]
Actually I'd consider proposing to use mambaforge - initial tests indicate that it is an order of magnitude faster

**Comment by jmwright** [comment]
So we would use the `mamba` command instead of `conda` in most cases? Mamba's documentation claims that it builds environments more reliably than conda, but I have no experience with it.

**Comment by adam-urbanczyk** [comment]
Yes, it is super fast but I don't have that much experience with it. Maybe it is too early to recommend.

**Comment by fedorkotov** [comment]
+1 for mamba from me.

I tried mamba on top of miniconda on Ubuntu 20.04 today. It is objectively **much** faster. 
My miniconda installation is not new so all packages were already in cache before my experiments.
I created two new environments. Installed cq-editor and cadquery in both (sequentially, not in parallel). I used conda in first environment and mamba in second. Execution time was measured with standard [time](https://man7.org/linux/man-pages/man1/time.1.html) command.

```bash
time conda install -y -c cadquery -c conda-forge cq-editor=master cadquery=master
...
real    1m53.195s
user    1m29.013s
sys     0m4.164s
```
```bash
time mamba install -y -c cadquery -c conda-forge cq-editor=master cadquery=master
...
real    0m20.962s
user    0m17.418s
sys     0m2.050s
```
